core/src/main/java/org/elasticsearch/monitor/os/OsProbe.java

```diff
@@ -191,6 +191,11 @@ private String readSingleLine(final Path path) throws IOException {
     // pattern for lines in /proc/self/cgroup
     private static final Pattern CONTROL_GROUP_PATTERN = Pattern.compile("\\d+:([^:,]+(?:,[^:,]+)?):(/.*)");
 
+    // this property is to support a hack to workaround an issue with Docker containers not mounting the cgroups hierarchy inconsistently
```

Comment:
should this be "not mounting...consistently" or "mounting...inconsistently"? But I would think not the current double negative.